frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -55,6 +55,21 @@ export const ParentSelect: React.FC<IParentSelect> = ({
   /** controls the multi selections displayed to the user */
   const [multiSelections, setMultiSelections] = React.useState<any>([]);
 
+  const headerAbbrvCheck = (options: SelectOptions, cutOffLength: number, parent: any) => {
```

Comment:
This is going to be removed and we are going to adjust the size of the options menu to contain all the agency labels, just going to to do that as a separate PR